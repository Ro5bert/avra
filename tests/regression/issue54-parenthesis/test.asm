.device atmega168
.macro DEST
	.dw @0
.endmacro
nop
KEYLOOP:
	DEST(KEYLOOP)
.if(1)
nop
.endif
