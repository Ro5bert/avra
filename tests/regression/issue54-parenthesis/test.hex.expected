:020000020000FC
:06000000000001000000F9
:00000001FF
